website/blog/2024-06-24-release-1.11.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Podman Desktop 1.11 Release
+description: Podman Desktop 1.11 has been released!
+slug: podman-desktop-release-1.11
+authors: cdrage
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.11/light-mode.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.11 Release! 🎉
+
+![Podman-desktop-1-11-hero](img/podman-desktop-release-1.11/banner.png)
+
+This release introduces:
+
+- **Light mode!**: Our most-requested feature is here! Try out our new experimental light mode in the settings.
+- **macOS Rosetta support for Apple Silicon**: Build AMD64 binaries at near-native speeds as ARM64 builds.
+- **Kubernetes improvements**: Check out our new Node and Volume pages for Kubernetes.
+- **Improved UI**: In addition to our light mode, you'll notice updates to our container listing page.
+- **Increased manifest support**: You'll now see images grouped together if built with a manifest.
+
+Podman Desktop 1.11 is now available. [Click here to download it](/downloads)!
+
+<!--truncate-->
+
+---
+
+## Release Details
+
+### Light Mode 💡
+
+We're excited to announce the arrival of our most-requested feature.. light mode! You can now enjoy a brighter interface by toggling the light mode option in the settings section. Please note, this feature is currently experimental as we continue to optimize the rest of Podman Desktop for this new mode.
+
+![Podman-desktop-1-11-hero](img/podman-desktop-release-1.11/light_mode.png)
+
+### macOS Rosetta Support for Apple Silicon
+
+Podman Desktop 1.11 proudly introduces support for macOS Rosetta, facilitating seamless integration with Apple Silicon. This update, part of Podman 5.1, allows users to enable or disable Rosetta support directly through the Podman Settings. With this enhancement, running AMD64 image builds or containers achieves speeds that are nearly identical to those experienced on ARM64 architectures. For more details on the implementation, visit the [GitHub podman pull request](https://github.com/containers/podman/pull/21670).
+
+### Kubernetes Enhancements
+
+In our latest release, we've enhanced Kubernetes functionality by adding node and volume listings. This update aims to improve your Kubernetes management experience within Podman Desktop.
+
+![kubernetes enhancements](img/podman-desktop-release-1.11/nodes.png)
+
+![kubernetes enhancements](img/podman-desktop-release-1.11/pvc.png)
+
+### User Interface Improvements
+
+Take a moment to appreciate the upgraded user interface of Podman Desktop. You'll find that the container listing section, among others, has received significant enhancements, elevating your overall user experience.
+
+### Enhanced Manifest Support
+
+We've made strides in how we handle manifest images. Now, when building a multi-arch image, manifest images are grouped together, simplifying the process and enhancing efficiency.
```

Comment:
'Now, when building a multi-arch image, manifest images are grouped together, simplifying the process and enhancing efficiency.' is a bit long with multiple commas. Maybe  'Multi-arch images are now grouped under the manifest, simplifying the process and enhancing efficiency.'?